tortoise/ninja_tortoise.go

```diff
@@ -120,7 +121,7 @@ func NewNinjaTortoise(layerSize int, blocks Mesh, hdist int, log log.Log) *ninja
 
 func (ni *ninjaTortoise) evictOutOfPbase() {
 	wg := sync.WaitGroup{}
-	if ni.pBase.Layer() <= ni.hdist {
+	if ni.pBase == ZeroPattern || ni.pBase.Layer() <= ni.hdist {
 		return
 	}
 
```

Comment:
For the line below (definition of 'window'), wouldn't it be better to make sure that  window is at least 0 (pbase=>hdist)?